src/topp/FileFilter.cpp

```diff
@@ -135,7 +135,42 @@ class TOPPFileFilter :
   }
 
 private:
-  static bool checkPeptideIdentification_(BaseFeature& feature, const bool remove_annotated_features, const bool remove_unannotated_features, const StringList& sequences, const StringList& accessions, const bool keep_best_score_id, const bool remove_clashes)
+  static bool sequenceIsWhiteListed_(const AASequence& peptide_hit_sequence, 
+                                     const StringList& whitelist, 
+                                     const String& sequence_comparison_method) {
+    //loop over all sequence entries of the StringList
+    for (StringList::const_iterator seq_it = whitelist.begin(); seq_it != whitelist.end(); ++seq_it)
+    {
+      if (sequence_comparison_method == "substring") {
```

Comment:
see coding convention... brackets go in new line

Reply:
oh snap, yes, thanks. I have resolved all brackets/whitespace issues I could spot now.